src/frontend/staff-portal/src/app/components/jj-workbench/jj-dispute-digital-case-file/jj-dispute-digital-case-file.component.ts

```diff
@@ -54,6 +54,14 @@ export class JJDisputeDigitalCaseFileComponent implements OnInit, AfterViewInit
       this.subscription = this.data$.subscribe(data => {
         this.dataSource.data = data;
         // this.dataSource.data = this.sampleData();
+ 
+        // load dataFilters (DCF tab) in session for page reload
+        let __dataFilter = sessionStorage.getItem("dataFilters-DCF");
```

Comment:
not a good habit to store inside session

Reply:
Okay. What's the alternative?

Reply:
ngrx